docs/books/incus_server/10-automating.md

```diff
@@ -61,7 +61,7 @@ Your entry will look like this:
 
 Save your changes and exit.
 
-This will create a log in incusadmin's home directory called "refreshlog" which will give you knowledge of whether your process worked or not. Very important!
+This will create a log-in incusadmin's home directory called "refreshlog," which will give you knowledge of whether your process worked or not. Very important!
 
```

Comment:
It's actually creating a "log", not a "log-in"... Perhaps there's a better way to word it so that it isn't confusing?  Let me know.